app/src/main/java/openfoodfacts/github/scrachx/openfood/views/BottomNavigationBehavior.java

```diff
@@ -0,0 +1,59 @@
+package openfoodfacts.github.scrachx.openfood.views;
+
+import android.support.annotation.NonNull;
+import android.support.design.widget.BottomNavigationView;
+import android.support.design.widget.CoordinatorLayout;
+import android.support.v4.view.ViewCompat;
+import android.view.View;
+
+/**
+ * Created by IMLV on 17-03-2018.
+ */
+
+public class BottomNavigationBehavior extends CoordinatorLayout.Behavior<BottomNavigationView> {
+
+    private int height;
+
+    @Override
+    public boolean onLayoutChild(CoordinatorLayout parent, BottomNavigationView child, int layoutDirection) {
+        height = child.getHeight();
+        return super.onLayoutChild(parent, child, layoutDirection);
+    }
+    /**
+     * this class is helper class for ProductActivity.java for enabling hiding bottomnavigation view on scroll
+     * This will hide bottom navigation view on up scroll
+     * will show on down scroll
+     */
+
```

Comment:
Could this documentation be moved to above line 13?

Reply:
@ross-holloway94  updated documentation  😃